emcc.py

```diff
@@ -3493,18 +3493,33 @@ def un_src(self):
     """Use this if you want to modify the script and need it to be inline."""
     if self.src is None:
       return
-    self.inline = '''
-          var script = document.createElement('script');
-          script.src = "%s";
-          document.body.appendChild(script);
-''' % self.src
+    if settings.EXPORT_ES6:
+      template = '''
+        import("./%s").then(exports => exports.default(Module))
+      '''
+    else:
+      template = '''
+            var script = document.createElement('script');
+            script.src = "%s";
+            document.body.appendChild(script);
+      '''
+    self.inline = template % self.src
```

Comment:
Maybe use the new f-strings things.  Put an `f` right before the string and use `{self.src}` in the string body.

Reply:
Nice, TIL.